develop/tutorials/articles/240-javascript-module-loaders/03-using-external-libraries.markdown

```diff
@@ -92,10 +92,11 @@ these steps to use your library:
 
 +$$$
 
-**Note:** As of @product-ver@, it's possible to hide Liferay's AMD Loader
-through @product@'s System Settings. Open the Control Panel, navigate to 
-*Configuration* &rarr; *System Settings*, and click *JavaScript Loader* under 
-the *Foundation* tab. Finally, uncheck the `expose global` option.
+**Note:** As of @product-ver@ GA4 (when released), it's possible to hide 
```

Comment:
Yes. We should mention all the relevant release info. So, instead of saying as of @product@ GA4, we should  say as of Liferay Portal CE GA4 and Liferay DXP fix pack 8 etc., it's possible to hide...